apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/CcwRifLoadJob.java

```diff
@@ -421,6 +394,43 @@ public void close() throws Exception {
     closer.finish();
   }
 
+  /**
+   * Retrieves the distinct set of all timestamps from each non-synthetic load in the 'Incoming'
+   * folder.
+   *
+   * @return set of timestamps
+   */
+  private Set<Instant> getAllNonSyntheticManifestTimestamp() {
+    return dataSetQueue
+        .readAllIncomingManifests(S3_PREFIX_PENDING_DATA_SETS)
+        .filter(id -> !id.manifestId().isFutureManifest())
+        .map(id -> id.manifestId().getTimestamp())
+        .collect(Collectors.toSet());
+  }
+
+  /**
+   * Returns the flattened list of manifests from the manifest lists.
+   *
+   * @param finalManifestLists final manifest lists from the 'Incoming' folder
+   */
```

Comment:
```suggestion
   * @param finalManifestLists final manifest lists from the 'Incoming' folder
   * @return set of manifests from all final manifest lists in 'Incoming'
   */
```

Looks like this javadoc is missing a `@return`.